queries.sql

```diff
@@ -0,0 +1,68 @@
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species is null;
+
```

Comment:
- [ ] Here after setting the `species` column, we have to verify the change was made, commit and also verify the change persist after commit